lang/pt_BR.json

```diff
@@ -0,0 +1,22 @@
+{
+    "VISION5E.Blindsense": "Sentido às Cegas",
+    "VISION5E.Burrowing": "Escavação",
+    "VISION5E.DetectEvilAndGood": "Detectar o Bem e o Mal",
+    "VISION5E.DetectMagic": "Detectar Magia",
+    "VISION5E.DetectPoisonAndDisease": "Detectar Veneno e Doença",
+    "VISION5E.DetectThoughts": "Detectar Pensamentos",
+    "VISION5E.DevilsSight": "Visão do Diabo",
+    "VISION5E.DivineSense": "Sentido Divino",
```

Comment:
Yeah, will right in that, make more sense. But **Visão do Diabo** is **Visão Diabólica** in the official D&D5e book in pt-BR.

Reply:
Is it "Senso" in the official book tough? In the translation module is "Sentido" (https://gitlab.com/fvtt-brasil/dnd5e/-/blob/master/dnd5e_pt-BR/lang/pt-BR/compendium/dnd5e.classfeatures.json?ref_type=heads#L186). If it's "Senso" also, I can add that to so that it is picked up by the automation.

Changed to "Visão Diabólica".

Reply:
In the book is "Sentido Divino". That's the feature from Paladin, right? If it is, is "Sentido Divino", but if you could add "Senso" and "Sentido", maybe the confusion will not be a problem.